src/gtk/webview_webkit2.cpp

```diff
@@ -950,7 +950,11 @@ bool wxWebViewWebKit::IsBusy() const
 
 void wxWebViewWebKit::SetEditable(bool enable)
 {
+#if WEBKIT_CHECK_VERSION(2, 8, 0)
     webkit_web_view_set_editable(m_web_view, enable);
+#else
+    g_object_set(m_web_view, "editable", enable ? TRUE : FALSE, NULL);
```

Comment:
It looks like the `editable` property is 2.8+ also:
https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView--editable

Looks like setting editable may not be possible on < 2.8.